### PR TITLE
add on update to services table in allocation details

### PR DESCRIPTION
**Pull request by drewbailey**
Adds On Update value to services table

image

### CODE REVIEW

File: ui/app/templates/allocations/allocation/index.hbs [hunk 1]
```diff
@@ -173,6 +174,7 @@
               <td data-test-service-name>{{row.model.name}}</td>
               <td data-test-service-port>{{row.model.portLabel}}</td>
               <td data-test-service-tags>{{join ", " row.model.tags}}</td>
+              <td data-test-service-onupdate>{{row.model.onUpdate}}</td>
```
Comment (drewbailey) [inline ui/app/templates/allocations/allocation/index.hbs:177]: I was unsure about this data tag and guessed

Comment (DingoEatingFuzz) [reply ui/app/templates/allocations/allocation/index.hbs:177]: The data tags are arbitrary, so this is fine. There is another step though (see later comment).

File: ui/tests/acceptance/allocation-detail-test.js [hunk 2]
```diff
@@ -240,6 +240,7 @@ module('Acceptance | allocation detail', function(hooks) {
 
       assert.equal(renderedService.name, serverService.name);
       assert.equal(renderedService.port, serverService.portLabel);
+      assert.equal(renderedService.onUpdate, serverService.onUpdate);
```
Comment (DingoEatingFuzz) [inline ui/tests/acceptance/allocation-detail-test.js:243]: This `onUpdate` property is associated with the above `data-test` attribute you added, but it unfortunately doesn't happen for free.

You'll need to also add this `onUpdate` property [to the page object](https://github.com/hashicorp/nomad/blob/main/ui/tests/pages/allocations/detail.js#L92-L98). That should also make the tests pass.

**Review by DingoEatingFuzz — CHANGES_REQUESTED** [review]
Nice clean implementation ✨ You have a failing test, hopefully my comments will help you sort that out.

File: ui/tests/acceptance/allocation-detail-test.js (hunk 2 above)
Comment (backspace) [reply ui/tests/acceptance/allocation-detail-test.js:243]: I aborted my review as I was just saying the same thing, otherwise 👍 nice! 😀

**Review by DingoEatingFuzz — APPROVED** [review]
:shipit:

**Pull request merged**